tests/configure-test-env.sh

```diff
@@ -53,7 +53,7 @@ fi
 
 # Update tinytex
 echo ">>>> Configuring TinyTeX environment"
-if [ -z $GH_TOKEN ] && [ -n $(command -v gh) ]
+if [ -z $GH_TOKEN ] && [[ -n $(command -v gh) ]]
```

Comment:
I think the safest way if to write this way 
```suggestion
if [[ -z $GH_TOKEN && -n $(command -v gh) ]]
```

So many versions that I was not sure which one to use : http://mywiki.wooledge.org/BashFAQ/031

Hopefully this woks on Mac

Reply:
I will give that a shot.